### PR TITLE
added bullets for unsupported networking features

### PR DESCRIPTION
**Pull request by daschott**
Updating docs so that people are aware of unsupported features.

### CODE REVIEW

**Comment by daschott** [comment]
@JMesser81 please LGTM.

File: virtualization/windowscontainers/container-networking/network-drivers-topologies.md [hunk 1]
```diff
@@ -19,23 +19,24 @@ In addition to leveraging the default 'nat' network created by Docker on Windows
   > Multiple NAT networks are supported if you have the Windows 10 Creators Update installed!
 
 - **transparent** – containers attached to a network created with the 'transparent' driver will be directly connected to the physical network through an *external* Hyper-V switch. IPs from the physical network can be assigned statically (requires user-specified ``--subnet`` option) or dynamically using an external DHCP server. 
+  > Note: Due to the below requirement, connecting your container hosts over a transparent network is not supported on Azure VMs.
```
Comment (JMesser81) [inline virtualization/windowscontainers/container-networking/network-drivers-topologies.md]: Can you ping Aanand on this? If he doesn't respond, leave it as is. But I think there are some VM instances which can support transparent mode (requires MAC spoofing on Hyper-V host)

**Comment by JMesser81** [comment]
@daschott - After pinging Aanand, let me know and I'll merge.

File: virtualization/windowscontainers/container-networking/network-drivers-topologies.md (hunk 1 above)
Comment (daschott) [reply virtualization/windowscontainers/container-networking/network-drivers-topologies.md]: @aanandr Are there any VM instances on Azure that support transparent mode networking?

**Comment by PleachiM** [comment]
I've heard transparent networking could be a feature in future (earlier this year) on Windows Server 2019? Is this correct? It's a huge problem for us in our Azure environment.

**Pull request merged**